python/dask_cudf/dask_cudf/expr/_expr.py

```diff
@@ -0,0 +1,205 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import functools
+
+from dask_expr._cumulative import CumulativeBlockwise, TakeLast
+from dask_expr._groupby import (
+    DecomposableGroupbyAggregation,
+    GroupbyAggregation,
+)
+from dask_expr._reductions import Var
+from dask_expr._shuffle import DiskShuffle
+
+from dask.dataframe.core import _concat
+
+from dask_cudf.groupby import (
+    _finalize_gb_agg,
+    _groupby_partition_agg,
+    _tree_node_agg,
+)
+
+##
+## Custom expression patching
+##
+
+
+class PatchCumulativeBlockwise(CumulativeBlockwise):
```

Comment:
Dask-expr's `CumulativeBlockwise` assumes positional arguments that are true for pandas, but not cudf.

Reply:
Yes - You've probably noticed that many of these "patches" can also be replced with fixes/features in cudf. It probably makes sense to open an issue and link it in a comment for each of these patches so it is easier to figure out when a patch can be removed.

Reply:
linked relevant issue.